docker/init_scripts/init

```diff
@@ -165,4 +189,7 @@ while true; do
 	watchdog_process_pid python worker
 	# Start scheduler if we dont have a corresponding PID file
 	watchdog_process_pid python scheduler
+
+  # check for died processes every 5 seconds
+  sleep 5
```

Comment:
I moved sleep to the end of the loop because it does not make sense to wait for 5 seconds when script is started instead of starting of all child processes.

Reply:
Good call!